app/javascript/components/get-version.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
+
+var lastValue = null;
+
+const getVersion = gql`
+  {
+    schemaSha
+  }
+`;
+
+export const Version = () => (
+  <Query query={getVersion}>
```

Comment:
Does this `Query` refresh the page on a stop server/edit HEROKU_SLUG_COMMIT/start server?

Reply:
This query will only trigger a refresh after mounting